dotCMS/src/main/java/com/dotcms/contenttype/business/uniquefields/extratable/ExtraTableUniqueFieldValidationStrategy.java

```diff
@@ -0,0 +1,178 @@
+package com.dotcms.contenttype.business.uniquefields.extratable;
+
+import com.dotcms.business.CloseDBIfOpened;
+import com.dotcms.business.WrapInTransaction;
+import com.dotcms.content.elasticsearch.business.ESContentletAPIImpl;
+import com.dotcms.contenttype.business.*;
+import com.dotcms.contenttype.business.uniquefields.UniqueFieldValidationStrategy;
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotcms.util.CollectionsUtils;
+import com.dotcms.util.JsonUtil;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.languagesmanager.model.Language;
+import com.dotcms.contenttype.model.field.Field;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UtilMethods;
+import com.google.common.annotations.VisibleForTesting;
+import com.liferay.portal.model.User;
+import org.postgresql.util.PGobject;
+
+import static com.dotcms.util.CollectionsUtils.list;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+
+/**
+ * {@link UniqueFieldValidationStrategy} that check the unique values using a SQL Query and a Extra table.
+ * This is the same extra table created here {@link com.dotmarketing.startup.runonce.Task241007CreateUniqueFieldsTable}
+ */
+public class ExtraTableUniqueFieldValidationStrategy extends UniqueFieldValidationStrategy {
```

Comment:
Rename to DBUniqueFieldValidationStrategy

Reply:
done https://github.com/dotCMS/core/pull/30466/files#diff-217758731836e15813d5ad2e85a398e4d52317cbbb9334d5961895f9d7883851R43